.goreleaser.yaml

```diff
@@ -0,0 +1,41 @@
+# yaml-language-server: $schema=https://goreleaser.com/static/schema.json
+project_name: imagebuilder
+before:
+  hooks:
+    - go mod tidy
+    - go generate ./...
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    ignore:
+      - goos: windows
+        goarch: arm64
+    main:
+      ./carpenter.go
+archives:
+  - id: my-archive
+    name_template: "{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}"
+    format_overrides:
+      - goos: windows
+        format: zip
+    files:
+      - LICENSE
+      - README.md
+dockers:
+  - dockerfile: Dockerfile
+    image_templates:
+      - ghcr.io/arcalot/arcaflow-plugin-image-builder:{{ .Tag }}
+      - ghcr.io/arcalot/arcaflow-plugin-image-builder:{{ .Major }}
+      - ghcr.io/arcalot/arcaflow-plugin-image-builder:latest
+checksum:
+    name_template: 'checksums.txt'
+changelog:
+  sort: asc
+  filters:
+    exclude:
+      - '^docs:'
+      - '^test:'
```

Comment:
If you are going to come back to this file, it is missing a trailing newline character.